src/domain/pricing/pricing-form/sections/configuration.tsx

```diff
@@ -0,0 +1,199 @@
+import { PriceList } from "@medusajs/medusa"
+import clsx from "clsx"
+import { useAdminCustomerGroups } from "medusa-react"
+import React, { useState } from "react"
+import { Controller } from "react-hook-form"
+import DatePicker from "../../../../components/atoms/date-picker/date-picker"
+import TimePicker from "../../../../components/atoms/date-picker/time-picker"
+import Switch from "../../../../components/atoms/switch"
+import Select from "../../../../components/molecules/select"
+import Accordion from "../../../../components/organisms/accordion"
+import { usePriceListForm } from "../form/pricing-form-context"
+import { ConfigurationField } from "../types"
+
+type ConfigurationProps = {
+  priceList?: PriceList
+}
+
+const checkForEnabledConfigs = (
+  config: Record<ConfigurationField, unknown>
+): string[] => {
+  const enabledConfigs: string[] = []
+
+  if (config.customer_groups) {
+    enabledConfigs.push(ConfigurationField.CUSTOMER_GROUPS)
+  }
+  if (config.starts_at) {
+    enabledConfigs.push(ConfigurationField.STARTS_AT)
+  }
+  if (config.ends_at) {
+    enabledConfigs.push(ConfigurationField.ENDS_AT)
+  }
+
+  return enabledConfigs
+}
+
+const Configuration: React.FC<ConfigurationProps> = () => {
+  const { customer_groups, isLoading } = useAdminCustomerGroups()
```

Comment:
thought (non-blocking): we probably need to implement some mechanism to handle pagination. Current core implementation only returns 10 customer groups, so we might run into issues sooner than elsewhere (i.e: regions). 

Found this [project](https://www.npmjs.com/package/react-select-async-paginate) that might help us implement this with `react-select`. I personally prefer we track this in a separate ticket and instead focus on getting this PR in 😄 . What do you think?

Reply:
I was also thinking that we need to handle this somehow, the project you linked looks like a great candidate! But I agree we should do that in a separate PR 😄